tcms/rpc/tests/test_testexecution.py

```diff
@@ -334,6 +334,38 @@ def test_update_with_non_existing_assignee_username(self):
                 "assignee": 'nonExistentUsername'
             })
 
+    def test_update_with_numeric_case_text_version(self):
+        initial_execution_case_text_version = self.case_run_1.case_text_version
+        case = self.rpc_client.TestCase.update(self.case_run_1.case.pk, {
+            'summary': 'This was updated',
+            'text': 'new TC text',
+        })
+        self.case_run_1.case.refresh_from_db()
+        updated_case_text_version = self.case_run_1.case.history.latest().history_id
+        self.assertNotEqual(initial_execution_case_text_version, updated_case_text_version)
+        execution = self.rpc_client.TestExecution.update(self.case_run_1.pk, {
```

Comment:
this line starts the second phase of the test, so add an empty line between line 345 and 346